### ISSUE TITLE
Deepfill2 model trained on celeba-HQ

### ISSUE THREAD

**Issue by cmyyy** [JiahuiYu/generative_inpainting#240]
Hello,@JiahuiYu.
I'm curious about the performance of deepfill2 on celeba-HQ, when the mask is regular, so could you share the pretrained model?Or would you mind sending me the inpaintng results when the hole is rectangular with the size of 128*128 and located in the middle of the mask?Here is my e-mail:luminouscmy@163.com
Thx!

**Comment by JiahuiYu** [comment]
Hi,

See issue #62 , we don't have a plan to release pre-trained model so far.

But, if you want to submit publications that try to compare with our results, we can help run results on our side and send the results back. In that case, you will need to prepare masks, inputs, file lists, and a description on why you need these results (for research purpose only).

**Comment by cmyyy** [comment]
Reason:
I have made some modifications based on your model and want make a comparison with your result.
Mask:
![center_mask_256](https://user-images.githubusercontent.com/26576058/55372016-c3032600-5532-11e9-9d83-1db482f44267.jpg)
Inputs: 
256*256 celeba-HQ images resized from 1024*1024. 
File lists:
[validation_shuffled.flist.tar.gz](https://github.com/JiahuiYu/generative_inpainting/files/3032096/validation_shuffled.flist.tar.gz)

**Comment by cmyyy** [comment]
@JiahuiYu , is it OK?

**Comment by JiahuiYu** [comment]
Let me run these when I have time. Do you think a few examples from validation set should be ok for your case?

**Comment by cmyyy** [comment]
emmm, I am afraid not as i need to calculate PSNR.
